xml/art_sle_ha_install_quick.xml

```diff
@@ -788,6 +794,139 @@ softdog                16384  1</screen>
        <keycap function="control"/><keycap>C</keycap>
       </keycombo>.
    </para>
+   </sect2>
+
+   <sect2 xml:id="sec-ha-inst-quick-test-with-cluster-script">
+    <title>Testing with the &test.script; Command</title>
+    <para>
+     The command &cmd.test.script; runs standardized tests
+     for a cluster.
+     It triggers cluster failures and verifies configuration to find
+     problems. Before you use your cluster in production, it is
+     recommended to use this command to make sure everything works as expected.
+    </para>
+    <para>
+     The command supports the following checks:
+    </para>
+    <itemizedlist>
+     <listitem>
+      <formalpara>
+       <title>Environment Check <option>-e</option>/<option>--env-check</option></title>
+       <para>
+        This test checks:
+       </para>
+      </formalpara>
+      <itemizedlist>
+       <listitem>
+        <para> Are host names resolvable? </para>
+       </listitem>
+       <listitem>
+        <para>
+         Is the time service enabled and started?
+        </para>
+       </listitem>
+       <listitem>
+        <para>
+         Has the current node configured watchdog?
+        </para>
+       </listitem>
+       <listitem>
+        <para>
+         Is the firewalld service started and are cluster related ports
+         open?
+        </para>
+       </listitem>
+      </itemizedlist>
+     </listitem>
+     <listitem>
+      <formalpara>
+       <title>Cluster State Check <option>-c</option>/<option>--cluster-check</option></title>
+       <para> Checks different states and services of the cluster. This test checks:</para>
+      </formalpara>
+      <itemizedlist>
+       <listitem>
+        <para>
+         Are cluster services (&pace;/&corosync;) enabled and running?
+        </para>
+       </listitem>
+       <listitem>
+        <para>
+         Is &stonith; enabled? It also checks, whether &stonith; related
+         resources are configured and started.
+         If you configured SBD, is the SBD service started?
+        </para>
+       </listitem>
+       <listitem>
+        <para>
+         Have clusters quorum? Shows current DC nodes and nodes which are online,
```

Comment:
Do clusters have quorum? (better check with Liam, too).

Reply:
@lproven what do you think? Should I better write `Do cluster nodes have quorum?` Or maybe there is even a better alternative? Thanks!

Reply:
I suggest:
> Does the cluster have a quorum?

Reply:
"Does the cluster have a quorum?"
or
"Is there a quorum of cluster nodes?"